railties/test/application/assets_test.rb

```diff
@@ -156,28 +148,10 @@ class User < ActiveRecord::Base; raise 'should not be reached'; end
       assert_no_file_exists("#{app_path}/public/assets/something.else-*.css")
     end
 
-    test "precompile something.js for directory containing index file" do
```

Comment:
Curious what you're planning to do with the sprockets assets tests

Reply:
My idea is to change tests to be compatible with Propshaft and remove what is not. Most of them are related to the fact that Propshaft can't compile `.erb` files.

Reply:
I don't think we should lose the sprockets tests, since we're not deprecating/removing it afaik